app/steps/medicaid/insurance_current_type.rb

```diff
@@ -1,27 +1,33 @@
 # frozen_string_literal: true
 
 module Medicaid
-  class InsuranceCurrentType < ManyMembersStep
+  class InsuranceCurrentType < Step
     step_attributes(
       :insurance_type,
-      :members,
+      :member_id,
     )
 
-    def insured_members_requesting_insurance
-      members.select(&:requesting_health_insurance).select(&:insured)
+    def valid?
+      if member_has_insurance? && no_insurance_type_provided?
+        errors.add(:insurance_type, "Please select a plan")
+        false
+      else
+        true
+      end
     end
 
     private
 
-    def validate_household_member(member)
-      if member.requesting_health_insurance? && member.insured?
-        unless member.insurance_type.present?
-          member.errors.add(
-            :insurance_type,
-            "Please select a plan",
-          )
-        end
-      end
+    def member_has_insurance?
+      member.requesting_health_insurance? && member.insured?
```

Comment:
if this were not true for a member, would we ever get to this step for them anyway?

Reply:
Don't know. I had just moved the logic from where it had been previously at line 17.